lms/envs/common.py

```diff
@@ -1601,6 +1601,7 @@
     #  (b) refactor their functionality out of the Blocklike objects and into the edx-platform block runtimes.
     LmsBlockMixin,
     InheritanceMixin,
+    ResourceTemplates,
```

Comment:
I don't think we need this change in LMS, because templates are only used to build new blocks in Studio. Try removing it, and see if tests still pass.